app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.jsx

```diff
@@ -1,6 +1,8 @@
+import Parser from 'html-react-parser';
 import classNames from 'classnames/bind';
+import AAIcon from '../img/aa_activated-inline.svg';
```

Comment:
Please move these icons to the `common/img`

Reply:
Fixed.